tools/ansible/roles/dockerfile/files/launch_awx_web.sh

```diff
@@ -15,6 +15,4 @@ fi
 
 set -e
 
-wait-for-migrations
```

Comment:
I want to document this change, because it is deceptively unrelated to the intent of the rest of the change.

This is an unrelated kind of bug fix - because the task container's obligation in this scenario is to serve the "AWX is upgrading" page, and it obviously can't do that if the container hangs out waiting for migrations to finish before serving. So web container should always go straight to serving.

Not true for task container.